src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -2132,7 +2132,7 @@ private function addUidSection(ArrayNodeDefinition $rootNode, callable $enableIf
                     ->children()
                         ->enumNode('default_uuid_version')
                             ->defaultValue(6)
-                            ->values([6, 4, 1])
+                            ->values([7, 6, 4, 1])
```

Comment:
I guess we should also add 8 as an allowed value here.

Reply:
Our factory does not know how to generate a v8 uuid AFAICT (as the spec does not explain how they are generated), so it would not work

Reply:
Right :man_facepalming: